### DISCUSSION TITLE
Plugins for the project manager

### DISCUSSION THREAD

**Discussion by Miskler** [godotengine/godot-proposals#7003] [Editor]
It would be convenient to be able to customize the project manager to your liking and complement its functionality.

**Comment by me2beats** [comment]
See also https://github.com/godotengine/godot-proposals/issues/1482
I imagine you could create your own UI for project manager, add new features like 5 stars instead of just 'favorites' etc

**Comment by Calinou** [comment]
- This is already being tracked in https://github.com/godotengine/godot-proposals/issues/1482.
